server/routes/auth.routes.js

```diff
@@ -136,22 +132,19 @@ function getGovernmentEmail(emails) {
 function createUser(loginGovUser) {
   logger.log('info', 'Creating new Login.gov user', {
     email: loginGovUser.email,
-    office: getOfficeFromEmail(loginGovUser.email),
     tag: 'createUser'
   })
 
   let now = new Date();
   let date = (now.getMonth() + 1) + "-" + now.getDate() + "-" + now.getFullYear();
-  let office = getOfficeFromEmail(loginGovUser.email);
-
+  const user_email = gov_email || loginGovUser.email
   let user_data = {
     'firstName': loginGovUser.given_name || null,
     'lastName': loginGovUser.family_name || null,
     'email': user_email,
     'password': null,
     'agency': grabAgencyFromEmail(user_email),
```

Comment:
Please add all the Agency email you found into the AGENCY_LOOKUP in the config.

For instance:
```
"army" : "Department of the Army",
"navy": "Department of the Navy",
etc.
```

Reply:
removed all these constants from the file